html/utilities/polyfills/ObjectPrepend.js

```diff
@@ -1,6 +1,6 @@
 // Source: https://github.com/jserz/js_piece/blob/master/DOM/ParentNode/prepend()/prepend().md
 (function (arr) {
-  arr.forEach((item) => {
+  arr.forEach(function (item) {
```

Comment:
Looks like we are changing the original Spark package here. This needs to be added to release notes as a bug fix.

Reply:
added